mewarpx/tests/diags/chkpoint00005/Level_0/Bx_fp_H

```diff
@@ -0,0 +1,41 @@
+2
```

Comment:
All these checkpoint files should go in `mewarpx/tests/test_files/` inside an appropriately named folder please.